### PR TITLE
Updated language selector

### PR DESCRIPTION
**Pull request by Raffox97**
* Added missing string in Italian for the new language selector
* Fixed wrong casing for language names in the language selector

### CODE REVIEW

File: web/src/helpers/translation.js [hunk 1]
```diff
@@ -44,20 +44,20 @@ exports.getFullZoneName = function getFullZoneName(zoneCode) {
...
-  { shortName: 'fr', name: 'français' },
-  { shortName: 'it', name: 'italiano' },
+  { shortName: 'es', name: 'Español' },
+  { shortName: 'fr', name: 'Français' },
+  { shortName: 'it', name: 'Italiano' },
   { shortName: 'ja', name: '日本語' },
   { shortName: 'nl', name: 'Nederlands' },
-  { shortName: 'pl', name: 'polski' },
-  { shortName: 'pt-br', name: 'português (Brazilian)' },
+  { shortName: 'pl', name: 'Polski' },
```
Comment (jarek) [inline web/src/helpers/translation.js]: I can't speak for other languages, but I can tell you for sure this is wrong - we don't capitalize language names in Polish as is done in English or German.

Comment (jld3103) [reply web/src/helpers/translation.js]: And why did you remove pt-br?

Comment (jarek) [reply web/src/helpers/translation.js]: pt-br is below, the display in Github is misleading because I attached my comment to the pl line

**Comment by Raffox97** [comment]
I'm closing that pull request for now, since I thought it's better for every language, to decide for it's own.

**Pull request closed without merging**